### PR TITLE
docs(README): Remove smart module

### PR DESCRIPTION
**Pull request by mhwelander**
Documentation update only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,17 +137,17 @@ Once Prisma Client was generated, you can import in your code and send queries t
 You can import and instantiate Prisma Client as follows:
 
 ```ts
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from "@prisma/client";
```
Comment (janpio) [inline README.md:140]: unrelated and unwanted change?

Comment (mhwelander) [reply README.md:140]: Ah - unwanted. My formatter did that. Will revert.

**Review by Akxe — CHANGES_REQUESTED** [review]
There is a lot of unrelated changes. A lot of quotation marks changed to double and semicolons were added. Since Prisma does not use either I suggest reverting the changes, only keeping changes to the 2 comments.

File: README.md (hunk 1 above)
Comment (janpio) [reply README.md]: There are many of those - just commented on this one.

**Review by janpio — CHANGES_REQUESTED** [review]
Unwanted code formatting changes need to be undone, then this is good to merge 👍

**Comment by mhwelander** [comment]
Prettified!

**Review by janpio — APPROVED** [review]

**Pull request merged**